packages/projection-typeorm/test/operators/storeNftMetadata.test.ts

```diff
@@ -447,3 +449,41 @@ describe('storeNftMetadata', () => {
     expect(['string', 'undefined'].includes(typeof file.name)).toBe(true);
   });
 });
+
+describe('willStoreNftMetadata', () => {
+  it('returns true if there are nftMetadata', () => {
+    expect(
+      willStoreNftMetadata({
+        cip67: {
+          byAssetId: {},
+          byLabel: {}
+        },
+        nftMetadata: [{} as ProjectedNftMetadata]
+      })
+    ).toBeTruthy();
+  });
+
+  it('returns true if there are cip67 metadata', () => {
```

Comment:
These are the tokens without metadata that can potentially have reference NFTs minted in a previous block
```suggestion
  it('returns true if there are cip67 tokens', () => {
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1200/commits/54c0c852e728eee3480f1244000a8165135b5455